core/src/main/java/io/grpc/internal/KeepAliveManager.java

```diff
@@ -45,7 +45,7 @@
  */
 public class KeepAliveManager {
   private static final SystemTicker SYSTEM_TICKER = new SystemTicker();
-  private static final long MIN_KEEPALIVE_DELAY_NANOS = TimeUnit.MINUTES.toNanos(1);
+  private static final long MIN_KEEPALIVE_DELAY_NANOS = TimeUnit.SECONDS.toNanos(10);
```

Comment:
Done, as a separate commit. It doesn't really apply to what I was trying to do in any existing commit. It also required renaming `keepAliveDelayInNanos` to `keepAliveTimeInNanos`. That does make `nextKeepaliveTime` slightly confusing, but not enough to worry me much.